### PR TITLE
chore(ci-cd): changes to make the repo public

### LINKED ISSUE #19: Make the repository public
**Is your feature request related to a problem? Please describe.**
Make the repository public

**Describe the solution you'd like**
Repository to be publicly available and follow all the open source standards.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by yeshamavani**
GH-19

## Description

Chnages to make the repository public

Fixes #19

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/tenant-management-service/package.json [hunk 1]
```diff
@@ -1,10 +1,18 @@
 {
-  "name": "tenant-management-service",
+  "name": "@sourceloop/tenant-management-service",
   "version": "0.0.1",
-  "description": "tenant-mgmt-service",
+  "description": "Tenant Management Service for multi-tenant SaaS applications",
```
[the remark this answers is not in the document]
Comment (yeshamavani) [reply services/tenant-management-service/package.json]: done

File: services/tenant-management-service/package.json [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
   "keywords": [
     "loopback-microservice",
-    "loopback"
+    "loopback",
+    "saas",
+    "tenant",
+    "tenant-management",
+    "tenant-service",
+    "control plane",
```
[the remark this answers is not in the document]
Comment (yeshamavani) [reply services/tenant-management-service/package.json]: done

File: services/subscription-service/package.json [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+  "description": "Subscription management service for multi-tenant SaaS applications",
   "keywords": [
     "loopback-microservice",
-    "loopback"
+    "loopback",
+    "saas",
+    "subscription",
+    "subscription-management",
+    "subscription-service",
+    "control plane",
```
[the remark this answers is not in the document]
Comment (yeshamavani) [reply services/subscription-service/package.json]: done

File: services/subscription-service/package.json [hunk 4]
```diff
@@ -1,10 +1,18 @@
 {
-  "name": "subscription-service",
+  "name": "@sourceloop/subscription-service",
   "version": "0.0.1",
-  "description": "subscription-service",
+  "description": "Subscription management service for multi-tenant SaaS applications",
```
[the remark this answers is not in the document]
Comment (yeshamavani) [reply services/subscription-service/package.json]: done

File: services/subscription-service/package.json [hunk 5]
```diff
@@ -1,10 +1,18 @@
 {
-  "name": "subscription-service",
+  "name": "@sourceloop/subscription-service",
```
[the remark this answers is not in the document]
Comment (yeshamavani) [reply services/subscription-service/package.json]: done

**Review by samarpan-b — APPROVED** [review]

File: LICENSE [hunk 6]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [2020-2024] [SourceFuse Technologies]
```
[the remark this answers is not in the document]
Comment (yeshamavani) [reply LICENSE]: done

**Pull request merged**